hucache.http/Cache.fs

```diff
@@ -4,11 +4,16 @@ open hucache.http.caches.postgres
 open hucache.http.types
 open hucache.http.github
 
-let lookup (key: IssueKey) : FullPayload option =
+let lookupIssue (key: IssueKey) : FullPayload option =
     match loadIssue key with
     | Some payload -> Some(payload)
     | None -> 
         let issue = fetchIssue key
         match issue with 
         | Some i -> Some((storeIssue (key, i)))
         | None -> None
+
+let lookupIssues (key: RepoKey) : FullPayload list = 
+    match loadIssues key with 
+    | [] -> []
+    | i -> i 
```

Comment:
Is there more to be done here eventually? Otherwise you could just `let lookupIssues = loadIssues` for now.